Mods/intact_mods.txt

```diff
@@ -19,4 +19,10 @@ TG   L or I
 MT   Unlocalized
 MM   -113.08406397853
 CF   C-6 H-11 O-1 N-1
-//
\ No newline at end of file
+//
+ID   Sulfate Adduct
+TG   Any
+PP   Anywhere.
+MM   97.967377
+MT   Adduct
+CF   H-2 O-4 S-1
```

Comment:
Need to add // at the end of this file for the last modification to be read, I think.